Makefile

```diff
@@ -248,14 +248,18 @@ test-nightly-server: build-nightly-server
 
 ### Check Rules: static checks, quality tools ###
 
-check: check-shell-scripts
+check: check-shell-scripts check-yaml
 .PHONY: check
-
 # rule requires shellcheck and find to run
 check-shell-scripts: $(filter $(ALT_BIN)%,$(SHELLCHECK))
 	$(SHELLCHECK) -P tests/ -eSC2181 -fgcc $$(find $(ROOT_DIR) -name "*.sh")
 .PHONY: check-shell-scripts
 
+
+check-yaml: $(filter $(ALT_BIN)%,$(YAMLLINT_CMD))
```

Comment:
Are we expecting `make check-yaml` to fail when _**yamllint**_ command is not available on the system? Because right now _hack/install-tool.sh_ is unaware of `--yamllint` option and fails.

In GitHub CI _**yamllint**_ is present by default as _/opt/pipx_bin/yamllint_.

Reply:
Hrm. I wonder if we have another case of a disappearing commit because I swear I remember there being a commit to add `--yamllint` at one point. But good catch regardless!

Reply:
This is very confusing, indeed, I am also sure it was there before, after all, the checks all passed in the PR previously... unsure what to do about it... 🤷🏼

Reply:
> In GitHub CI _**yamllint**_ is present by default as _/opt/pipx_bin/yamllint_.

and that's why it passes on GitHub.

We'll have to resurrect the change within _hack/install-tools.sh_ script to install **_yamllint_** if not present on the system.

Reply:
@anoopcs9   even if it was not needed for the CI passing, I re-added the `--yamllint` switch to the install-tool script

Reply:
@anoopcs9   even if it was not needed for the CI passing, I re-added the `--yamllint` switch to the install-tool script

Reply:
@phlogistonjohn  - FYI